components/activity_and_insights/insights/least_active_channels/least_active_channels_item/least_active_channels_item.tsx

```diff
@@ -43,6 +43,37 @@ const LeastActiveChannelsItem = ({channel, actionCallback}: Props) => {
         return iconToDisplay;
     }, [channel]);
 
+    let timeMessage = (
+        <FormattedMessage
+            id='insights.leastActiveChannels.lastActivity'
+            defaultMessage='Last activity: {time}'
+            values={{
+                time:
+                (
+                    <Timestamp
+                        value={channel.last_activity_at}
+                        units={[
+                            'now',
+                            'minute',
+                            'hour',
+                            'day',
+                            'week',
+                            'month',
+                        ]}
+                        useTime={false}
+                    />
+                ),
+            }}
+        />
+    );
+    if (channel.last_activity_at === 0) {
+        timeMessage = (
+            <FormattedMessage
+                id='insights.leastActiveChannels.lastActivityNone'
+                defaultMessage='No activity'
```

Comment:
This needs `values.time` too, no?

Reply:
No because we are only showing `No activity` and there is no time displayed